### PR TITLE
Rename Precompute Balance vars to be more descriptive

### PR DESCRIPTION
**Pull request by 0xKiwi**
Rename Precompute Balances to:
```
pBal.ActiveCurrent
pBal.ActivePrevEpoch 
pBal.CurrentEpochAttested
pBal.CurrentEpochTargetAttested  
pBal.PrevEpochAttested
pBal.PrevEpochTargetAttested
pBal.PrevEpochHeadAttested
```

### CODE REVIEW

File: beacon-chain/core/epoch/precompute/type.go [hunk 1]
```diff
@@ -40,21 +40,21 @@ type Validator struct {
...
 // Pre computing and storing such record is essential for process epoch optimizations.
 type Balance struct {
-	// CurrentEpoch is the total effective balance of all active validators during current epoch.
-	CurrentEpoch uint64
-	// PrevEpoch is the total effective balance of all active validators during prev epoch.
-	PrevEpoch uint64
-	// CurrentEpochAttesters is the total effective balance of all validators who attested during current epoch.
-	CurrentEpochAttesters uint64
-	// CurrentEpochTargetAttesters is the total effective balance of all validators who attested
+	// CurrentEpochBal is the total effective balance of all active validators during current epoch.
```
Comment (terencechain) [inline beacon-chain/core/epoch/precompute/type.go:43]: `Balance.CurrentEpochBal` sounds a bit odd...

How about `Balance.ActiveCurrentEpoch` ? The active/non active part was the source of the confusion anyway

Comment (0xKiwi) [reply beacon-chain/core/epoch/precompute/type.go:43]: Some of the rename also came from it just being `p.CurrentEpoch`, maybe I'll try renaming that `p` to something else as well.

Comment (terencechain) [reply beacon-chain/core/epoch/precompute/type.go:43]: I think the problem is `p`, we can change that to something better like `b`, `balance`... etc

File: beacon-chain/rpc/beacon/validators.go [hunk 2]
```diff
@@ -816,13 +816,13 @@ func (bs *Server) getValidatorParticipationUsingOldArchival(
...
-		EligibleEther: p.PrevEpoch,
-		VotedEther:    p.PrevEpochTargetAttesters,
+		EligibleEther: p.ActivePrevEpoch,
+		VotedEther:    p.PrevEpochTargetAttested,
 	}
 	participation.GlobalParticipationRate = float32(0)
 	// only divide if prevEpoch is non zero
-	if p.PrevEpoch != 0 {
-		participation.GlobalParticipationRate = float32(float64(p.PrevEpochTargetAttesters) / float64(p.PrevEpoch))
+	if p.ActivePrevEpoch != 0 {
```
Comment (terencechain) [inline beacon-chain/rpc/beacon/validators.go:824]: Rename `p` to `pBal` here too?

Comment (0xKiwi) [reply beacon-chain/rpc/beacon/validators.go]: Done! Nice catch

**Pull request merged**